public/JavaScript/gallery-dom.js

```diff
@@ -52,4 +53,20 @@ document.addEventListener('DOMContentLoaded', function () {
       radio.checked = false
     })
   }
+
+  function displayGalleryItemsNum () {
```

Comment:
You could make this testable by passing in the name of the class and use jsdom to simulate a structure and check that it returns correctly